src/utils/string.rs

```diff
@@ -55,9 +56,46 @@ pub fn take_lines<R: RangeArgument<usize>>(s: &str, range: R) -> String {
     }
 }
 
+/// Take anchored lines from a string.
+/// Lines containing anchor are ignored.
+pub fn take_anchored_lines(s: &str, anchor: &str) -> String {
+    lazy_static! {
+        static ref RE_START: Regex = Regex::new(r"ANCHOR:\s*(?P<anchor_name>[\w\d_-]+)").unwrap();
```

Comment:
Just a minor nit, I think the `\d` is redundant here (I believe `\w` already covers numbers).

Reply:
That's right! I'll remove it :+1: